test/authorization/service/role.service.test.ts

```diff
@@ -13,59 +18,71 @@ import { AuthenticationHelper } from '../../../src/authentication/authentication
 import { RedisCacheService } from '../../../src/cache/redis-cache/redis-cache.service';
 import { ConfigService } from '@nestjs/config';
 import GroupRole from '../../../src/authorization/entity/groupRole.entity';
-import Role from '../../../src/authorization/entity/role.entity';
 import { RoleService } from '../../../src/authorization/service/role.service';
 import RolePermission from '../../../src/authorization/entity/rolePermission.entity';
 import RoleCacheService from '../../../src/authorization/service/rolecache.service';
 import SearchService from '../../../src/authorization/service/search.service';
 import Group from '../../../src/authorization/entity/group.entity';
-const roles: Role[] = [
-  {
-    id: 'ae032b1b-cc3c-4e44-9197-276ca877a7f8',
-    name: 'Test1',
-  },
-];
-
-const permissions: Permission[] = [
-  {
-    id: '2b33268a-7ff5-4cac-a87a-6bfc4430d34c',
-    name: 'Customers',
-    label: 'Customers',
-  },
-];
+import { RoleRepository } from '../../../src/authorization/repository/role.repository';
+import { PermissionRepository } from '../../../src/authorization/repository/permission.repository';
+import Role from 'src/authorization/entity/role.entity';
+import { RoleNotFoundException } from '../../../src/authorization/exception/role.exception';
 
 describe('test Role Service', () => {
   let roleService: RoleService;
-  const rolesRepository = Substitute.for<Repository<Role>>();
-  const permissionRepository = Substitute.for<Repository<Permission>>();
+  let roleRepository: RoleRepository;
+  let permissionRepository: PermissionRepository;
   const groupRoleRepository = Substitute.for<Repository<GroupRole>>();
   const rolePermissionRepository = Substitute.for<Repository<RolePermission>>();
   const roleCacheService = Substitute.for<RoleCacheService>();
   const redisCacheService = Substitute.for<RedisCacheService>();
   const searchService = Substitute.for<SearchService>();
-  const connectionMock = Substitute.for<Connection>();
   const groupRoleQueryBuilder = Substitute.for<SelectQueryBuilder<GroupRole>>();
   const permissionQueryBuilder = Substitute.for<
     SelectQueryBuilder<Permission>
   >();
-  const rolesQueryBuilder = Substitute.for<SelectQueryBuilder<Role>>();
+
+  let createQueryBuilderMock: jest.Mock;
+  let getManyAndCountMock: jest.Mock;
+  let findOneByMock: jest.Mock;
+  let saveMock: jest.Mock;
+  let updateMock: jest.Mock;
+  let getManyMock: jest.Mock;
```

Comment:
Unused import.

Reply:
Will fix service related tests in a separate PR.